packages/schema/src/buildSchemaFromTypeDefinitions.ts

```diff
@@ -8,21 +8,22 @@ import { concatenateTypeDefs } from './concatenateTypeDefs';
 export function buildSchemaFromTypeDefinitions(
   typeDefinitions: ITypeDefinitions,
   parseOptions?: GraphQLParseOptions,
-  noExtensionExtraction?: boolean
+  noExtensionExtraction?: boolean,
+  assumeValidSchema?: boolean
 ): GraphQLSchema {
   const document = buildDocumentFromTypeDefinitions(typeDefinitions, parseOptions);
 
   if (noExtensionExtraction) {
-    return buildASTSchema(document);
+    return buildASTSchema(document, { assumeValid: assumeValidSchema });
   }
 
   const { typesAst, extensionsAst } = filterAndExtractExtensionDefinitions(document);
 
-  const backcompatOptions = { commentDescriptions: true };
-  let schema: GraphQLSchema = buildASTSchema(typesAst, backcompatOptions);
+  const buildAstOptions = { assumeValid: assumeValidSchema, commentDescriptions: true };
```

Comment:
When is it assumeValidSDL vs assumeValidSchema

Reply:
Or just assumeValid? Do you want to give options for both? I assume the assumeValid gets embedded in schema for later? Seems to also include assuming valid sdl, although that can be set separately?